go/cli/mcap/cmd/filter.go

```diff
@@ -416,6 +441,18 @@ usage:
 			0,
 			"messages with log times before nanosecond-precision timestamp will be included.",
 		)
+		startDate := filterCmd.PersistentFlags().String(
+			"start-date",
```

Comment:
nit `start-datetime`? Since it is both a date and time? Probably I'd even say that `start` is sufficient and can accept the start string as RFC3339 or the nanosecond variant. Not "real" reason that the program can't figure it out between those two formatss

Reply:
should `start` and `start-secs` accept an RFC3339 formatted timestamp in that world?

Reply:
`start-secs` I'd have a harder time making an argument for given it says "secs". `start` seems to be asking for flexible support input formats